lib/pengine/pe_output.c

```diff
@@ -202,7 +202,11 @@ static const char *
 get_cluster_stack(pcmk_scheduler_t *scheduler)
 {
     xmlNode *stack = get_xpath_object(XPATH_STACK, scheduler->input, LOG_DEBUG);
-    return (stack != NULL)? crm_element_value(stack, PCMK_XA_VALUE) : "unknown";
+
+    if (stack != NULL) {
+        return crm_element_value(stack, PCMK_XA_VALUE);
+    }
+    return PCMK_VALUE_UNKNOWN;
```

Comment:
Technically this is also arbitrary. `PCMK_OPT_CLUSTER_INFRASTRUCTURE` isn't a "select" type; we allow an arbitrary string.

In practice we only support "corosync" right now.